swift/aperture/Recorder.swift

```diff
@@ -1,13 +1,22 @@
 import AVFoundation
 
+enum ApertureError: Error {
+  case couldNotAddScreen
+  case couldNotAddMic
+  case couldNotAddOutput
```

Comment:
@LarsJK What's the convention for when to include an error message and not? Most of the Swift code I've seen does it like this.